ci-operator/step-registry/jboss-fuse/deploy-resources/jboss-fuse-deploy-resources-commands.sh

```diff
@@ -0,0 +1,369 @@
+EOF
+
+  oc create configmap mvn-settings -n ${1} --from-file=/tmp/settings.xml
+  oc create configmap test-properties -n ${1} --from-file=/tmp/test.properties
+}
+
+function create_foo_volumes()
+{
+  oc create -f - <<EOF
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: postgresql-persistent
+  labels:
+    app: postgresql-persistent
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 1Gi
+  accessModes:
+    - ReadWriteOnce
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Recycle
+  hostPath:
+    path: "/mnt"
+EOF
+
+oc create -f - <<EOF
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: persistent-fuse-1
+  labels:
+    type: local
+    app: xpaas-qe
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 20Gi
+  accessModes:
+    - ReadWriteOnce
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Recycle
+  hostPath:
+    path: "/mnt"
+EOF
+
+  oc create -f - <<EOF
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: persistent-xpaas-qe
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 19Gi
+  selector:
+    matchLabels:
+      app: xpaas-qe
+  storageClassName: manual
+  volumeMode: Filesystem
+status:
+  accessModes:
+    - ReadWriteMany
+    - ReadWriteOnce
+  capacity:
+    storage: 19Gi
+EOF
+}
+
+function create_nginx() {
+  NGINX_DOMAIN="nginx.${CONSOLE_URL#"https://console-openshift-console."}"
+  NGINX_HOST=${NGINX_DOMAIN//".org/"/.org}
+
+  oc import-image ${1}/nginx:latest --from=quay.io/packit/nginx-unprivileged --confirm -n ${1}
+  oc create -f - <<EOF
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: nginx-conf
+data:
+  default.conf: |
+    server {
+        listen       8080;
+        server_name  localhost;
+
+        #access_log  /var/log/nginx/host.access.log  main;
+
+        location / {
+            autoindex on;
+            root   /deployments/.m2/repository;
+            index  index.html index.htm;
+        }
+
+        #error_page  404              /404.html;
+
+        # redirect server error pages to the static page /50x.html
+        #
+        error_page   500 502 503 504  /50x.html;
+        location = /50x.html {
+            root   /deployments/.m2/repository;
+        }
+
+        # proxy the PHP scripts to Apache listening on 127.0.0.1:80
+        #
+        #location ~ \.php$ {
+        #    proxy_pass   http://127.0.0.1;
+        #}
+
+        # pass the PHP scripts to FastCGI server listening on 127.0.0.1:9000
+        #
+        #location ~ \.php$ {
+        #    root           html;
+        #    fastcgi_pass   127.0.0.1:9000;
+        #    fastcgi_index  index.php;
+        #    fastcgi_param  SCRIPT_FILENAME  /scripts;
+        #    include        fastcgi_params;
+        #}
+
+        # deny access to .htaccess files, if Apache's document root
+        # concurs with nginx's one
+        #
+        #location ~ /\.ht {
+        #    deny  all;
+        #}
```

Comment:
can we remove lines with comments, or is it needed for creating ConfigMap for nginx?